layers/+lang/clojure/packages.el

```diff
@@ -316,11 +316,12 @@ If called with a prefix argument, uses the other-window instead."
           "mrcn" 'cljr-clean-ns
           "mrcp" 'cljr-cycle-privacy
           "mrdk" 'cljr-destructure-keys
-          "mref" 'cljr-extract-function
           "mrec" 'cljr-extract-constant
+          "mref" 'cljr-extract-def
+          "mref" 'cljr-extract-function
```

Comment:
Oops, should be `mred`, thanks.

I'm on vacation now and can't change the PR so feel free to fix that during
merging.

Malchevskiy Misha <notifications@github.com> schrieb am Mi., 30. Sep. 2015
14:02:

> In layers/+lang/clojure/packages.el
> <https://github.com/syl20bnr/spacemacs/pull/2989#discussion_r40781541>:
>
> >            "mrec" 'cljr-extract-constant
> > +          "mref" 'cljr-extract-def
> > +          "mref" 'cljr-extract-function
>
> Both have the same binding?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/2989/files#r40781541>.
>